docs/en/ingest-management/release-notes/release-notes-8.19.asciidoc

```diff
@@ -0,0 +1,151 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.19.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.19.0 relnotes
+
+[[release-notes-8.19.0]]
+== {fleet} and {agent} 8.19.0
+
+Review important information about the {fleet} and {agent} 8.19.0 release.
+
+[discrete]
+[[security-updates-8.19.0]]
+=== Security updates
+
+{agent}::
+* Upgrade To Go 1.24.3. {elastic-agent-pull}8109[#8109]
+
+{fleet-server}::
+* Upgrade golang.org/x/net to v0.34.0 and golang.org/x/crypto to v0.32.0. {fleet-server-pull}4405[#4405]
+
+[discrete]
+[[new-features-8.19.0]]
+=== New features
+
+The 8.19.0 release Added the following new and notable features.
+
+{agent}::
+* Set replicas for Gateway Collector. {elastic-agent-pull}7011[#7011]
```

Comment:
Quite a few of the new attributes are misbehaving.
Preview: https://ingest-docs_bk_1834.docs-preview.app.elstc.co/guide/en/fleet/8.19/release-notes-8.19.0.html